backend/EasySpeak.Core/EasySpeak.Core.BLL/Services/LessonsService.cs

```diff
@@ -115,4 +119,46 @@ public async Task<LessonDto> CreateLessonAsync(NewLessonDto lessonDto)
 
         return _mapper.Map<LessonDto>(createdLesson);
     }
+
+    public async Task<TeacherStatisticsDto> GetTeacherLessonsStatisticsAsync()
+    {
+        var statistics = await _context.Lessons
```

Comment:
_context.Lessons.GroupBy(_ => 1).Select(el => new statsDTO {
TotalClasses = el.Count(l => l.CreatedBy == _auth...

}

I guess something like this can be done